zdm/tests/craco.py

```diff
@@ -0,0 +1,102 @@
+""" CRACO FRBs: This may move out of tests someday """
```

Comment:
My main comment is that this file *should* be redundant with my craco mc stats file, since the first step there is to set up CRACO and generate ZDM grids. I propose that these two files get merged into a single functionality at the next update. Let me know if you or I should do this.

Reply:
I've taken the initiative to bring together all the CRACO files in a `craco/` folder.